mlir/lib/Quantum/Transforms/merge_rotation.cpp

```diff
@@ -67,6 +70,10 @@ struct MergeRotationsPass : impl::MergeRotationsPassBase<MergeRotationsPass> {
         if (failed(applyPatternsAndFoldGreedily(targetfunc, std::move(patterns)))) {
             return signalPassFailure();
         }
+
+        auto stop = std::chrono::high_resolution_clock::now();
+        auto duration = std::chrono::duration_cast<std::chrono::microseconds>(stop - start);
+        llvm::errs() << "merge rotation pass runtime: " << duration.count() << " microseconds\n";
```

Comment:
@paul0403 quick question, I saw I was tagged for review but I assume this branch is not going to be merged into `main`? 😅

Reply:
No need to review, this is just for visibility, or if others want to use the compile time timing/plotting script (which I will add)